src/03-error-handling-async/index.test.ts

```diff
@@ -1,30 +1,55 @@
-// Uncomment the code below and write your tests
-// import { throwError, throwCustomError, resolveValue, MyAwesomeError, rejectCustomError } from './index';
+import {
+  throwError,
+  throwCustomError,
+  resolveValue,
+  MyAwesomeError,
+  rejectCustomError,
+} from './index';
 
 describe('resolveValue', () => {
   test('should resolve provided value', async () => {
-    // Write your test here
+    const value = 1;
+    const expected = value;
+
+    await expect(resolveValue(value)).resolves.toBe(expected);
   });
 });
 
 describe('throwError', () => {
   test('should throw error with provided message', () => {
-    // Write your test here
+    const message = 'Error message';
+    const expected = new Error(message);
```

Comment:
you dont need create new Error to provide in toThrow
this method cant use message: string as argument ``toThrow('Aaargh!');``